payload/recipes/ibexa_install.bash

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env bash
+
+cd $PROJECTMAPPINGFOLDER
+
+PHP="php"
+COMPOSER="$PHP -d memory_limit=-1 /usr/local/bin/composer"
+REPO=$1
+VERSION=$2
+INIT_DATA=$3
+DATABASE_PREFIXES=${DATABASE_PREFIXES:-DATABASE}
+CONSOLE="bin/console"
+
+for prefix in $DATABASE_PREFIXES
+do
+    DATABASE_NAME_VAR=${prefix}_NAME
+    DATABASE_HOST_VAR=${prefix}_HOST
+    DATABASE_USER_VAR=${prefix}_USER
+    DATABASE_PASSWORD_VAR=${prefix}_PASSWORD
+
+    # Wait for the DB
+    while ! mysqladmin ping -h"${!DATABASE_HOST_VAR}" -u"${!DATABASE_USER_VAR}" -p"${!DATABASE_PASSWORD_VAR}" --silent; do
+        echo -n "."
+        sleep 1
+    done
+    echo ""
+
+    mysql -h"${!DATABASE_HOST_VAR}" -u"${!DATABASE_USER_VAR}" -p"${!DATABASE_PASSWORD_VAR}" -e "CREATE DATABASE ${!DATABASE_NAME_VAR}"
+done
+
+echo "Installation Ibexa ($REPO - $VERSION) in the container"
+
+# Install ibexa/website-skeleton
+echo "Install ibexa/website-skeleton"
+$COMPOSER create-project --no-interaction ibexa/website-skeleton ezplatform $VERSION
+cd ezplatform
+# Copy nginx conf
+echo "Getting the NGINX config"
+wget https://github.com/ibexa/docker/archive/main.zip
+unzip main.zip
+mkdir -p doc
+cp -r docker-main/templates/nginx doc/
+rm -rf docker-main
+rm main.zip
+
+# Add .env.local to set database configuration
+echo "Add .env.local to set database configuration"
+echo "DATABASE_URL=\${DATABASE_PLATFORM}://\${DATABASE_USER}:\${DATABASE_PASSWORD}@\${DATABASE_HOST}:\${DATABASE_PORT}/\${DATABASE_NAME}?serverVersion=\${DATABASE_VERSION}" > ".env.local"
```

Comment:
Look like this part is missing support for `DATABASE_PREFIXES`

Reply:
I don't think so, that's not really managed by Ibexa anyway. So for the `install` that should be fine.
Actually the multi db for `install` does not have to be supported. It is there but I don't really remember why. Do you? (you did it ;))